manifests/bosh-manifest/deployments/010-bosh-template.yml

```diff
@@ -52,8 +52,6 @@ jobs:
       password: (( grab secrets.bosh_nats_password ))
 
     redis:
-      listen_address: 127.0.0.1
-      address: 127.0.0.1
```

Comment:
You can expect some reasonable default & that it is working. We don't specify address for anything else but NATS because it is compulsory property. Moreover, for some reason, redis and cc spec use different property names (address and listen_address). If you don't want to rely on defaults (btw. redis will likely be removed in our next BOSH upgrade), then you should add test that the both are same.